src/functions/template-tags/general.php

```diff
@@ -1380,11 +1380,15 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 			$excerpt = wp_trim_words( $excerpt, $excerpt_length, $excerpt_more );
 		}
 
-		// Remove all shortcode Content before removing HTML
-		if ( ! $allow_shortcode ) {
-			$excerpt = preg_replace( '#\[.+\]#U', '', $excerpt );
+		// If shortcode filter is enabled lets process them
+		if ( $allow_shortcode ) {
+			$excerpt = do_shortcode( $excerpt );
 		}
 
```

Comment:
I changed it so the shortcode is processed if the filter is true, do we want to change it so we always process now? I believe that could create support issues. Anytime we change how the display of the excerpts here it always create support so I would side with only processing shortcodes if the filter is active and otherwise strip them out.